plugins/statistics.koplugin/main.lua

```diff
@@ -72,9 +72,9 @@ local ReaderStatistics = Widget:extend{
     page_stat = {}, -- Dictionary, indexed by page (hash), contains a list (array) of { timestamp, duration } tuples.
     data = {
         title = "",
-        authors = "",
-        language = "",
-        series = "",
+        authors = "N/A",
+        language = "N/A",
+        series = "N/A",
```

Comment:
Untranslated on purposes, these are used as SQL search parameters, we want them to be language-agnostic.